applications/zpc/components/zwave_command_classes/src/zwave_command_class_meter_control.c

```diff
@@ -205,13 +205,21 @@ static sl_status_t zwave_command_class_meter_get(attribute_store_node_t node,
   attribute_store_node_t scale_node
     = attribute_store_get_first_parent_with_type(rate_type_node,
                                                  ATTRIBUTE(SCALE));
+  zwave_cc_version_t supporting_node_version
+    = zwave_command_class_get_version_from_node(node, COMMAND_CLASS_METER);
+
   meter_scale_t scale = 0;
   attribute_store_get_reported(scale_node, &scale, sizeof(scale));
 
   ZW_METER_GET_V5_FRAME *get_frame = (ZW_METER_GET_V5_FRAME *)frame;
   get_frame->cmdClass              = COMMAND_CLASS_METER_V6;
```

Comment:
not related to you pr, but I would expect to have v6 frame to align the cmd class version maybe different ones will work but I bring confusion IMHO

Reply:
V6 is only locally defined
https://github.com/SiliconLabs/UnifySDK/blob/db3a8e63e7ada18f6e0379f886d4e447107f1736/applications/zpc/components/zwave_command_classes/src/zwave_command_class_meter_control.c#L42

Reply:
Ok this should have been done differently, I'll see how to avoid this kind of duplication...

FTR at zwave we want to generate headers from xml file, unify should do same...